src/simulation.md

```diff
@@ -329,11 +329,13 @@ Analogous to the `door_supervisor`, a `lift_supervisor` [node](https://github.co
 A common use case is robots performing deliveries within facilities, so a `Delivery` task is configured into the `rmf_fleet_adapters`.
 In a delivery task, a payload is loaded onto the robot at one location (pickup waypoint) and unloaded at another (dropoff waypoint).
 The loading and unloading of the payload may be automated by a robot/workcell in the facility.
-To integrate these systems with RMF core systems, a set of [dispenser messages](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg) are defined.
-Despite their names, the messages are generalized for use by systems other than dispensers as well.
+To integrate these systems with RMF core systems, a set of [dispenser](https://github.com/osrf/rmf_core/tree/master/rmf_dispenser_msgs/msg) and [ingestor](https://github.com/osrf/rmf_core/tree/master/rmf_ingestor_msgs) messages are defined.
```

Comment:
Thanks for updating the documentation. Could we explicitly state what is meant by dispenser and ingestor resp? Dispenser is an automation equipment that handles loading of a payload onto the robot while ingestor handles unloading. Or something along those lines.

Reply:
Sure, just added that information in a separate PR.